src/lex-client.ts

```diff
@@ -48,10 +52,48 @@ export default class LexClient {
     return formattedString;
   }
 
+  // method for building a bot locale, version and alias
+  build = async (botId: string, botAlias: string, botLocale?: string): Promise<void> => {
+    const locale = botLocale || 'en_US';
+
+    if (this.verbose) {
```

Comment:
I like your use of a verbose flag for logging

Reply:
Thanks! helps me find out where I went wrong!